packages/react/src/themes/teams/colors.ts

```diff
@@ -122,7 +132,7 @@ export const primitiveColors: PrimitiveColors = {
   white: '#fff',
 }
 
-export const transparentColors = {
+export const transparentColors: TeamsTransparentColors = {
   silver: {
     100: undefined,
```

Comment:
just for the curiosity sake: these values that are `undefined` technically may be omitted from declarations, so that instead of 

```js
 yellow: {
    50: undefined,
    100: '#FBF6D9', // old message highlight color
    200: undefined,
    300: '#F9EC02', // old acc critical ufd icon color
    400: '#F8D22A', // old siteVariables.yellow, $app-yellow, yellow[900]
    500: '#FFB900', // old $bcast pre live color
    600: '#FFAA44', // new away presence
    700: undefined,
    800: undefined,
    900: undefined,
  },
```

we might define just what actually defined:

```js
 yellow: {
    100: '#FBF6D9', // old message highlight color
    300: '#F9EC02', // old acc critical ufd icon color
    400: '#F8D22A', // old siteVariables.yellow, $app-yellow, yellow[900]
    500: '#FFB900', // old $bcast pre live color
    600: '#FFAA44', // new away presence
  },
```

However, I might assume there is a reason why explicit `undefined` values are present - is it done for the sake of explicitness?

Reply:
Yes, exactly, this are defined as not defined in the designs as well, because we said that it will be more obvious that those are not defined for this color, instead of just missed. Also in the docs, we want to emphasize the same point, when showing the color palette.